config/webhook.yaml

```diff
@@ -87,11 +87,20 @@ spec:
       # high value that we respect whatever value it has configured for the lame duck grace period.
       terminationGracePeriodSeconds: 300
 ---
-apiVersion: v1
-kind: Secret
+apiVersion: sigstore.dev/v1alpha1
+kind: ClusterImagePolicy
```

Comment:
Kubernetes it terrible about installing CRDs and immediately instantiating them.

IMO the default behavior should not require an instance of the CRD to be installed.